gooddata-sdk/gooddata_sdk/utils.py

```diff
@@ -214,6 +229,24 @@ def mandatory_profile_content_check(profile: str, profile_content_keys: KeysView
 
 
 def profile_content(profile: str = "default", profiles_path: Path = PROFILES_FILE_PATH) -> dict[str, Any]:
+    """Get the profile content from a given file.
+
+    Args:
+        profile (str, optional):
+            Profile Name. Defaults to "default".
+        profiles_path (Path, optional):
+            File path for the profiles. Defaults to PROFILES_FILE_PATH.
```

Comment:
Check `good_pandas_profile_content` why the docstring differs for a profile.
I suggest using :

Profile name. Defaults to "default".